scripts/bulk-parse.js

```diff
@@ -0,0 +1,62 @@
+const parser = require('./parse-json.js');
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+
+
+const DEFAULT_INPUT_DIR = `./source/PCT`;
+const DEFAULT_OUTPUT_DIR = `./json`;
+
+const bulkParse = (input, output) => {
+    if (!fs.existsSync(input) ) {
+        throw new Error(`${input} directory could not be found or opened.`);
+    }
+
+    if (!fs.existsSync(output)) {
+        fs.mkdir(output);
+    } else {
+        fs.readdir(output, (err, files) => {
+            if (err) throw new Error(err);
```

Comment:
Couple things here:
 
 * Avoid the temptation to use "shortened" variable names, good variables should always be descriptive and readable. In this case I'd prefer `error` over `err`. 
 * `new Error(err)` is not necessary, it's already an error, and you're unwrapping the actual underlying error class. In this case `throw err` should suffice.